jsapp/js/components/common/tabs.tsx

```diff
@@ -0,0 +1,58 @@
+import React, {useState} from 'react';
+import styles from './tabs.module.scss';
+
+interface Tab {
+  label: string;
+  route: string;
+}
+
+export interface TabsProps {
+  tabs: Tab[];
+  selectedTab: string;
+  onChange: (route: string) => void;
+}
+
+export default function Tabs({tabs, selectedTab, onChange}: TabsProps) {
+  const [activeTab, setActiveTab] = useState(selectedTab);
+
+  const handleTabKeyDown = (event: React.KeyboardEvent<HTMLButtonElement>) => {
+    if (event.key === 'Tab') {
+      event.preventDefault();
+
+      const currentIndex = tabs.findIndex((tab) => tab.route === activeTab);
+
+      const nextIndex = (currentIndex + 1) % tabs.length;
+
+      setActiveTab(tabs[nextIndex].route);
+      onChange(tabs[nextIndex].route);
+    }
+  };
```

Comment:
This should probably change the active tab in response to the [left and right arrow keys](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role#keyboard_interaction) instead of the tab key.